packages/audio-element/.storybook/.babelrc

```diff
@@ -0,0 +1,15 @@
+{
+  "plugins": [
+    "@babel/plugin-syntax-dynamic-import",
+    "@babel/plugin-proposal-object-rest-spread"
```

Comment:
QUESTION: Is this JS? Do you want a trailing comma here?

Reply:
This was from the `open-wc` storybook template.. it's in [JSON5](https://json5.org) format, which can have trailing commas, but I haven't really touched this file at all.